src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1907,62 +1826,75 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         Set<String> studentsEmailInTeam = new HashSet<String>();
         if (role == UserRole.STUDENT) {
             student = studentsLogic.getStudentForEmail(courseId, userEmail);
-            List<StudentAttributes> studentsInTeam = studentsLogic
-                    .getStudentsForTeam(student.team, courseId);
+            List<StudentAttributes> studentsInTeam = studentsLogic.getStudentsForTeam(student.team, courseId);
             for (StudentAttributes teammates : studentsInTeam) {
                 studentsEmailInTeam.add(teammates.email);
             }
         }
-        
+
         InstructorAttributes instructor = null;
         if (role == UserRole.INSTRUCTOR) {
             instructor = instructorsLogic.getInstructorForEmail(courseId, userEmail);
         }
         
         Map<String, FeedbackResponseAttributes> relevantResponse = new HashMap<String, FeedbackResponseAttributes>();
```

Comment:
A suggestion would be to handle each part separately, questions -> responses -> comments and not trying to stick to what was done before, where one single method and entity was responsible for so many things